src/Compilers/Core/MSBuildTask/Shared/ManagedCompiler.cs

```diff
@@ -73,7 +73,7 @@ public string ChecksumAlgorithm
         }
         
         /// <summary>
-        /// An instrument flag that specifies instrumentation settings.
+        /// An instrument flag that specifies instrumentation kind.
```

Comment:
It is hard to tell whether this is intended to support a single kind of instrumentation, or multiple kinds. I does not help that this sentence is missing the article "the" (e.g. "the instrumentation kind") and also "kind" is singular, making it somewhat awkward and ambiguous.